cfme/tests/containers/test_ssl_providers.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from itertools import product
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.provider.openshift import OpenshiftProvider
+from utils import testgen
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+from utils import conf
+from cfme.web_ui import fill, flash
+from utils.providers import get_crud
+from utils.ssh import SSHClient
+from cfme.fixtures import pytest_selenium as sel
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8.0.3")]
```

Comment:
Don't really understand.. I do use a provider here and it must be above this version where this feature was introduced